### PR TITLE
[WIP]Data Going to Summary Chart

### PR DESCRIPTION
**Pull request by goldblatt**
Sending data to the summary chart. Known Issues: 

- Overall color of the header (how do we determine? need to get this for the status icon)
- Refactor some functions into utils instead of enums
- Use more enums across the codebase
- How to handle empty county data

### CODE REVIEW

File: src/components/LocationPage/ChartsHolder.tsx [hunk 1]
```diff
@@ -61,24 +63,36 @@ const ChartsHolder = (props: {
...
+
   return (
     <>
       {!projection ? (
         <NoCountyDetail countyId={props.countyId} stateId={props.stateId} />
       ) : (
         <ChartContentWrapper>
           <LocationPageHeader projections={props.projections} />
-          <SummaryStats stats={[]} />
+          {projection.isInferred && <SummaryStats stats={getChartSummarys(projection)} />}
```
Comment (igorkofman) [inline src/components/LocationPage/ChartsHolder.tsx:85]: this shouldn't gate on inferred, just on their being data in 
    const rt = this.primary.rt!;
    const testPositiveRate = this.primary.currentTestPositiveRate;
    const icuUtilization = this.primary.currentIcuUtilization;

File: src/components/LocationPage/ChartsHolder.tsx [hunk 2]
```diff
@@ -61,24 +63,36 @@ const ChartsHolder = (props: {
...
           <MainContentInner>
             <ChartHeader></ChartHeader>
-            <h1>Case growth</h1>
+            <h1>{ChartType.CASE_GROWTH_RATE}</h1>
```
Comment (igorkofman) [inline src/components/LocationPage/ChartsHolder.tsx:88]: It's kind of strange to use the value of the enum as the website copy.

Comment (goldblatt) [reply src/components/LocationPage/ChartsHolder.tsx:88]: bhahahahah yeah it isss

Comment (goldblatt) [reply src/components/LocationPage/ChartsHolder.tsx]: okay i made vars for this

**Pull request merged**